src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  width: 80%;
+
+  &__star {
+    height: $star-size;
+    width: $star-size;
+    background-image: url("/images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 .stars__star:nth-child(-n+4) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
Could you bring back selectors for 1-5 stars? The amount of active stars should be exactly the same as the value of `stars--N` modifier.

You could also do it with some sass loop if you want ;)